### ISSUE TITLE
Format naming photo

### ISSUE THREAD

**Issue by sh1man999** [kwon-evan/STLPRNet#5]
Hello, please tell me what to name the photo for training, what format is it?
What does it mean in the file name. Example CI0882LLE-3-1.jpg
Why is this "-3-1"?
Why lmdb is not used here?

**Comment by kwon-evan** [comment]
Hello, @sh1man999.
"-3-1" is for duplicate labels. the dataset will slice the first "-" in the filename and use it as the answer.
and the data I have is not lmdb. it is just a bunch of images with their label in filename. so I didnt use lmdb.

**Comment by sh1man** [comment]
can you tell me how to implement it via lmdb? and why didn't you use lmdb

**Comment by kwon-evan** [comment]
you can replace RawDataset to [LmdbDataset](https://github.com/clovaai/deep-text-recognition-benchmark/blob/e2117f2fb882b3c6085030500a260c113be27a63/dataset.py#L219).
i dont remember the details, but i think it was because i didnt want to install LMDB and convert the data to LMDB format.

**Comment by kwon-evan** [comment]
I forgot to mention the format of the dataset - I didn't use a public dataset. The data is a folder of images with labelled file names.
```
data/
 ├AA0000BB-0.jpg
 ├AA0000BB-1.jpg
 ├AA0000BB-2.jpg
  ...
```

**Comment by sh1man** [comment]
If I have license plates of different formats, rectangular and square, what imgH height should I use to train

**Comment by kwon-evan** [comment]
image size doesnt matter, it is automatically resized to (100, 32) during preprocessing in dataloader.

**Comment by sh1man** [comment]
what about square numbers?  Do you need to resize (100,100) and add pad?

**Comment by kwon-evan** [comment]
LPRNet takes as input a torch.tensor of size (100, 32). I've never tried a square, so if you have a square image, I recommend cropping it to the size of a licence plate. If you can't crop it, padding is also work too
